### ISSUE TITLE
bibliography style template

### ISSUE THREAD

**Issue by novica** [sdstrowes/Glasgow-Thesis-Template#3]
Hi, 

Is there a bibliography style template that can produce references as required by http://www.gla.ac.uk/services/library/subjectssupport/informationskills/referencing/#Harvard%20Guide ?

**Comment by sdstrowes** [comment]
Wondering if this is sufficient for Harvard style: https://github.com/sdstrowes/Glasgow-Thesis-Template/commit/2234aa7e000554921e2faca53b605de2f4670f2d

I used the IEEE transactions style mainly because I like it! I realise I've deviated from the style guide here.

**Comment by novica** [comment]
The thing is that apparently there is no one Harvard style. The university requires Anglia Ruskin's flavor: http://libweb.anglia.ac.uk/referencing/harvard.htm

So, if IEEE looks like this: www.ieee.org/documents/ieeecitationref.pdf then that's not the same as the above.

The in text citations are not a big issue, but the format of the bibliography with long / short first name, year in brackets or not... etc.

**Comment by sdstrowes** [comment]
Oh, I see what you mean.

The Harvard package (http://www.ctan.org/tex-archive/macros/latex/contrib/harvard/) seems to be carried by default in the texlive installation, and it has a bunch of .bst styles: \bibliographystyle{agsm}, \bibliographystyle{apsr}, etc.

Do any of those get close enough to Anglia Ruskin's flavour? If so, I'm happy to incorporate! If Anglia Ruskin deviates too far from these _and_ they don't publish their own .bst, then that's a pain.

**Comment by novica** [comment]
I asked and  they don't provide their .bst.

Googling I found this journal https://home.elementascience.org/for-authors/style-guide/ that has a reference to Anglia Ruskin and a style that they suggest http://endnote.com/downloads/style/elementa, but I'm new to Latex and don't know how to test that really.

So far I'm using Lyx and just trying to add whatever works in the preamble of the document. I sort of have a feeling that's not the best way to do it for larger projects.

**Comment by sdstrowes** [comment]
Right. What I'll aim to do is look at the Anglia Ruskin guidelines, the links you've provided, and incorporate something into the style. Fair warning, it'll be 1-2 weeks before I get to it proper!

**Comment by novica** [comment]
Thanks Stephen.

**Comment by novica** [comment]
Hi Stephen, 

I ended up using kluwer style with some tweaks over the year. So can you incorporate that one? I'll try to fine tune it afterwards. Thanks.

**Comment by azzamsa** [comment]
@novica Hi novica.

Do you have working 'Anglia Ruskin's flavor' style in your latex document ?
We can't find their .bst and still can't get it working.

Thanks in advance.

**Comment by novica** [comment]
Hi azzamsa,

No, at the time I didn't manage to make the Anglia Ruskin .bst.
Sometime along the way I moved to biblatex. It turned out to be easier to get needed formatting with it. 
But it was quite awhile ago, and I don't really recall the details.

**Comment by azzamsa** [comment]
I just come to Tex SE to ask this question. [This answer](https://tex.stackexchange.com/questions/446633/redefine-author-year-to-match-harvard-anglia-flavour/446641#446641) works great for me.

I am undergraduate student. I am not too familiar with scientific writing. Sorry for question. Is Harvard-angila flavor widely used ?, I don't find any question regarding it (in term of latex issue) just 2 of them in google result. I don't have any clue either why my institution use it.

Thank you.

**Comment by novica** [comment]
To be honest I have no idea how widely used is the Anglia Ruskin's flavor of the Harvard style. It is possible that they just provide a good guide of all possible citations and then other universities just point to their guide instead of creating their own. As far as I know this citation format is typical for social sciences. Legal studies use different format, history also. And then there is another format for science and technical writing.

**Comment by azzamsa** [comment]
> To be honest I have no idea how widely used is the Anglia Ruskin's flavor of the Harvard style. It is possible that they just provide a good guide of all possible citations and then other universities just point to their guide instead of creating their own. 

Emm, this maybe one of the reasons.

>. As far as I know this citation format is typical for social sciences. Legal studies use different format, history also. And then there is another format for science and technical writing.

TIL. So that another IT college here use IEEE format. I don't know why my IT college using this. Maybe I will ask the reason later.

Thanks lot for the answer, Novica. Thanks
